README.md

```diff
@@ -1,2 +1,8 @@
-# examples
+# Examples
 Example files using the neurobagel schema
+
+
+| Example                             | Description                                                                                                                                                                                                                                                                                                                                                           |
+|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| data-upload/example_synthetic.jsonld            | Sample output of `bagel pheno` command on the `synthetic` example inputs. Contains subject-level annotated phenotypic attributes obtained by essentially applying the data dictionary (.json) to the original tabular data (.tsv).                                                                                                                                    |
+| data-upload/pheno-bids-output/example_synthetic_pheno-bids.jsonld | Sample output of `bagel bids` command on the example_synthetic.jsonld and the [BIDS `synthetic` example dataset directory](https://github.com/bids-standard/bids-examples/tree/master/synthetic). Contains subject-level annotated imaging metadata (extracted directly from the BIDS dataset structure) on top of the subject-level annotated phenotypic attributes. |
```

Comment:
This looks good so far, but I would add info about the other files in this repo as well (e.g., descriptions of the TSV and JSON that are also in `data-upload/` from https://github.com/neurobagel/bagel-cli/pull/204/files), so a user can find exactly what they need. 🙂

I'm realizing that a table might not actually work best for this esp if we will add more examples in the future (messy diffs). How about something like this format instead:

### `data-upload/`
Examples of valid input files for Neurobagel harmonization tools, including resultant Neurobagel graph-ready dataset files.
- **example_synthetic.tsv**: valid phenotypic tabular data file (has `participant` and `session` IDs corresponding to the [`synthetic` bids-examples dataset](https://github.com/bids-standard/bids-examples/tree/master/synthetic))
- **example_synthetic.json**: valid phenotypic data dictionary for example_synthetic.tsv containing Neurobagel annotations
- **example_synthetic.jsonld**: example graph-ready subject data file for a purely phenotypic dataset. Obtained from running the [Neurobagel CLI](https://github.com/neurobagel/bagel-cli) ... (rest of description from table)

#### `pheno-bids-output/`

Example graph-ready data files containing both harmonized phenotypic and imaging (meta)data.

- **example_synthetic_pheno-bids.jsonld**: (description from table)

### `query-tool-results/`
Example downloadable query result TSV files from the [Neurobagel query tool](https://query.neurobagel.org).

Reply:
Good call @alyssadai